imagedephi/main.py

```diff
@@ -24,12 +27,31 @@ def imagedephi() -> None:
     "input-dir", type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path)
 )
 @click.argument(
-    "output-dir", type=click.Path(exists=True, file_okay=False, writable=True, path_type=Path)
+    "output-dir", type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path)
 )
-def run(input_dir: Path, output_dir: Path) -> None:
-    """Run in CLI-only mode."""
-    redact_images(input_dir, output_dir)
-    click.echo("Done!")
+@click.argument("override-rules", type=click.File("r"), required=False)
```

Comment:
This should be a `click.option`: https://click.palletsprojects.com/en/8.1.x/options/ , which will be more user-friendly than an optional positional argument.

Reply:
7178ba5